app/views/search/top_forty.html.erb

```diff
@@ -0,0 +1 @@
+<%= button_to "Discover Top 40 Movies" %>
```

Comment:
Looking at the wireframes, I think that our view pages for top 40 and movie search results are nested under /movies. This is what I was still confused on with the difference between Issue #7 and Issue #9